### PR TITLE
Optimize theme Pafat for the mobile view

### PR DESCRIPTION
**Pull request by myuki**
It is better for the mobile view to delete the button's `min-width`.

Before:

Before

After:

After

### CODE REVIEW

**Comment by Alkarex** [comment]
Thanks @Myuki 👍🏻 
I suggest you make the change in a `@media` section to only apply it when the width of the window is small.
Either in the existing one (see below), or in a new one to be applied even when the width is a bit larger.

https://github.com/FreshRSS/FreshRSS/blob/9ea3e7710ad54e76ff1f84bafb208da865816f93/p/themes/Pafat/pafat.css#L1131

**Comment by Frenzie** [comment]
Around 1280 and 1600 are other common breakpoints, presumably based on typical device screensizes.

**Comment by myuki** [comment]
> Thanks @Myuki 👍🏻
> I suggest you make the change in a `@media` section to only apply it when the width of the window is small.
> Either in the existing one (see below), or in a new one to be applied even when the width is a bit larger.
> 
> https://github.com/FreshRSS/FreshRSS/blob/9ea3e7710ad54e76ff1f84bafb208da865816f93/p/themes/Pafat/pafat.css#L1131

It's a better way. I will change later.

File: p/themes/Pafat/pafat.css [hunk 1]
```diff
@@ -1168,6 +1168,7 @@ a.btn {
 
 	.nav_menu .stick {
 		margin: 0 10px;
+		min-width: max-content;
```
Comment (Alkarex) [inline p/themes/Pafat/pafat.css]:
```suggestion
		min-width: 0;
```

Should achieve the same, while being lighter and more back-compatible

File: p/themes/Pafat/pafat.rtl.css [hunk 2]
```diff
@@ -1168,6 +1168,7 @@ a.btn {
 
 	.nav_menu .stick {
 		margin: 0 10px;
+		min-width: max-content;
```
Comment (Alkarex) [inline p/themes/Pafat/pafat.rtl.css]:
```suggestion
		min-width: 0;
```

**Comment by Alkarex** [comment]
@Myuki If you could be motivated enough to look at some of the other themes for this specific problem, please do so :-)
Thanks again 👍🏻

**Comment by myuki** [comment]
> 
> 
> @Myuki If you could be motivated enough to look at some of the other themes for this specific problem, please do so :-)
> Thanks again 👍🏻

I had checked other themes, no one use `min-width` as the same way. But some button is too big for mobile view.

**Comment by Alkarex** [comment]
Ok, thanks!

**Pull request merged**